### PR TITLE
APIS-4724 New manage API metadata page

### PR DESCRIPTION
**Pull request by adampridmore**

### CODE REVIEW

File: app/controllers/forms.scala [hunk 1]
```diff
@@ -381,9 +437,12 @@ object SubscriptionFieldsForm {
         mapping(
           "name" -> text,
           "description" -> text,
+          "sortDescription" -> text,
```
Comment (adampridmore) [inline app/controllers/forms.scala]: I think this is a type!
sortDescription
shortDescription

Comment (AndySpaven) [reply app/controllers/forms.scala]: :+1:

File: app/controllers/ManageSubscriptions.scala [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import domain.ApiSubscriptionFields.SubscriptionFieldValue
+import domain.ApiSubscriptionFields.SubscriptionFieldsWrapper
+import cats.data.NonEmptyList
+
+object ManageSubscriptions {
+  case class SubscriptionDetails(shortDescription: String, value: String)
+  case class ApiDetails(name: String, version: String, subsValues: NonEmptyList[SubscriptionDetails])
+
+  def toDetails(value: String): String = {
+    if (value == "") "None" else value
```
Comment (peteslater-ee) [inline app/controllers/ManageSubscriptions.scala]: Is using .isEmpty on the String better than the == ""?

Comment (AndySpaven) [reply app/controllers/ManageSubscriptions.scala]: Sure

**Pull request merged**